### PR TITLE
NMS-13885: Tuning broker docs

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13885

### CODE REVIEW

**Review by dino2gnt — DISMISSED** [review]
Glorious.

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 1]
```diff
@@ -9,9 +9,9 @@ from being published to the broker.
 
 == Monitoring the ActiveMQ broker using the Karaf shell
 
-The `opennms:activemq-stats` command available via the _Karaf shell_ can be used to show statistics about the embedded broker:
+The `opennms:activemq-stats` command available via the Karaf shell can be used to show statistics about the embedded broker:
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:12]:
```suggestion
Use the `opennms:activemq-stats` command available via the Karaf shell to show statistics about the embedded broker:
```

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 2]
```diff
@@ -37,28 +37,30 @@ Users associated with the `ADMIN` role can read, write or create any queue or to
...
 
-See the `authorizationPlugin` section in `$OPENNMS_HOME/etc/opennms-activemq.xml` for details.
+See the `authorizationPlugin` section in `$\{OPENNMS_HOME}/etc/opennms-activemq.xml` for details.
 
-== Multi-tenancy with _{page-component-title}_ and ActiveMQ
+== Multi-tenancy with {page-component-title} and ActiveMQ
 
-The queue names used by _{page-component-title}_ are prefixed with a constant value.
-If many _{page-component-title}_ are configured to use the same broker, then these queues would end up being shared amongst the instances, which is not desired.
+The queue names used by {page-component-title} are prefixed with a constant value.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:44]:
```suggestion
The queue names {page-component-title} uses are prefixed with a constant value.
```

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 3]
```diff
@@ -37,28 +37,30 @@ Users associated with the `ADMIN` role can read, write or create any queue or to
...
+If many {page-component-title} are configured to use the same broker, then these queues would end up being shared amongst the instances, which is not desired.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:45]:
```suggestion
If many {page-component-title} are configured to use the same broker, then these queues would end up being shared among the instances, which is not desired.
```

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 4]
```diff
@@ -37,28 +37,30 @@ Users associated with the `ADMIN` role can read, write or create any queue or to
...
+.`$\{MINION_HOME}/etc/custom.system.properties`.
+[source, properties]
 ----
-echo 'org.opennms.instance.id=MyNMS' > "$MINION_HOME/etc/custom.system.properties"
+org.opennms.instance.id=MyNMS
 ----
+
 CAUTION: If you change the instance ID setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix.
-The configuration can be modified with the file `$OPENNMS_HOME/etc/opennms-activemq.xml`:
+The configuration can be modified with the file `$\{OPENNMS_HOME}/etc/opennms-activemq.xml`:
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:63]:
```suggestion
Modify the configuration with the file `$\{OPENNMS_HOME}/etc/opennms-activemq.xml`:
```

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 5]
```diff
@@ -110,11 +125,10 @@ If you want to verify that a specific location is operating correctly make sure
...
 
-[source]
+[source, karaf]
 ----
-opennms:poll -l LOCATION -c org.opennms.netmgt.poller.monitors.IcmpMonitor 127.0.0.1
+opennms:poll -l LOCATION -c org.opennms.netmgt.poller.monitors.IcmpMonitor 127.0.0.1 <1>
 ----
-
-TIP: Replace `LOCATION` in the command above with the name of the location you want to test.
+<1> Replace `LOCATION` in the command above with the name of the Monitoring Location you want to test.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:134]:
```suggestion
<1> Replace `LOCATION` in the command above with the name of the monitoring location you want to test.
```

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 6]
```diff
@@ -10,25 +10,40 @@ Find available configuration parameters for Kafka here:
...
+[source, properties]
+----
+org.opennms.instance.id=MyNMS
+----
+
 == Tips for Kafka
 
-TIP: For Kafka RPC, the number of partitions should always be greater than the number of minions at a location.
-     When there are multiple locations, partitions >= max number of minions at a location.
+* For Kafka RPC, the number of partitions should always be greater than the number of minions at a location.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:34]:
```suggestion
* For Kafka RPC, the number of partitions should always be greater than the number of Minions at a location.
```

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 7]
```diff
@@ -10,25 +10,40 @@ Find available configuration parameters for Kafka here:
...
+When there are multiple locations, partitions >= max number of minions at a location.
 
-TIP: By default, Kafka RPC supports buffers greater than >1MB by splitting large buffer into chunks of 900KB(912600).
-     Max buffer size (900KB, by default) can be configured by setting org.opennms.core.ipc.rpc.kafka.max.buffer.size ( in bytes).
+* By default, Kafka RPC supports buffers greater than >1MB by splitting large buffer into chunks of 900KB.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:37]:
```suggestion
* By default, Kafka RPC supports buffers greater than >1MB by splitting a large buffer into chunks of 900KB.
```

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 8]
```diff
@@ -10,25 +10,40 @@ Find available configuration parameters for Kafka here:
...
+Max buffer size (900KB, by default) can be configured by setting `org.opennms.core.ipc.rpc.kafka.max.buffer.size` (in bytes).
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:38]:
```suggestion
Configure max buffer size (900KB, by default) by setting `org.opennms.core.ipc.rpc.kafka.max.buffer.size` (in bytes).
```

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 9]
```diff
@@ -10,25 +10,40 @@ Find available configuration parameters for Kafka here:
...
 
-TIP: Default time to live (time at which request will expire) is 20000 msec (20sec).
-     It can be changed by configuring system property `org.opennms.core.ipc.rpc.kafka.ttl` in msec.
+* Default time to live (time at which request will expire) is 20000 msec (20sec).
+It can be changed by configuring system property `org.opennms.core.ipc.rpc.kafka.ttl` in msec.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:41]:
```suggestion
To change it, configure the system property `org.opennms.core.ipc.rpc.kafka.ttl` in msec.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few minor changes.

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc (hunk 7 above)
Comment (mmahacek) [reply docs/modules/reference/pages/configuration/tuning-kafka.adoc:37]:
```suggestion
* By default, Kafka RPC supports buffers greater than >1MB by splitting them into chunks of 900KB.
```

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc (hunk 8 above)
Comment (mmahacek) [reply docs/modules/reference/pages/configuration/tuning-kafka.adoc:38]:
```suggestion
Configure max buffer size (900KB, by default) by setting `org.opennms.core.ipc.rpc.kafka.max.buffer.size` to the desired size in bytes.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**